scripts/scenario_03_edgeofspace.lua

```diff
@@ -171,21 +178,28 @@ function init()
 	---------------------------------------------------------------------------
 	---------------------------------------------------------------------------
 	
-	--Central Command sends us to repair E.O.S. Scope
+
+
+	--Central Command sends us to investigate the issues with E.O.S. Scope --Expanded text to attempt to explain why Apollo is shuttling this data around physically
 	Central_Command:sendCommsMessage(Player, [[Apollo, Come in.
 	Our Edge-of-space telescope has been experiencing malfunctions over the course of the past few days. We expect the cause to be a mechanical failure, but we want you to take a look. 
 	
-	The E.O.S Scope is right on the border of Kraylor space, so make sure to maintain contact and keep up long-range scans. Dock with the E.O.S Scope and perform whatever repairs are necessary, then report back with the cause.
-	
+	The E.O.S Scope is right on the border of Kraylor space, so make sure to maintain contact and keep up long-range scans. 
+
+	Dock with the E.O.S Scope and investigate the damage, then return to Central Command to report your findings. Transmission of this report via standard communications channels is considered too dangerous given the already delicate nature of our treaty with the Kraylor.

```

Comment:
Hopefully this firmly establishes the expectation that the Apollo must return to Central Command.